app/dashboard/templates/profiles/tab_theme.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n static %}
+{% block settings_content %}
+
+<form method="POST" id="settings" class="col-md-8 m-auto">
+    <div class="form-group">
```

Comment:
indent fix

Reply:
@thelostone-mc Fixed those issues. Please review it.